cypress/e2e/IPToLocationAPI/documentation.cy.js

```diff
@@ -0,0 +1,70 @@
+import { IPTOLOCATIONAPI_BASE_URL } from '../../resources/data';
+
+describe('IP To Location API Documentation page', () => {
+    before(() => {
+      cy.visit(IPTOLOCATIONAPI_BASE_URL); 
+    });
+    
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+  
+    it('2. Test to check "Documentation" tab visibility', () => {
+        cy.componentVisiblityCheck('.d-md-flex > .d-none');
+    });
+  
+    it('3. Test to click "Documentation" tab', () => {
+        cy.intercept('GET', '/marketplace/currency_data-api/tabs/api_docs').as('apiRequest');
+        cy.get('.d-md-flex > .d-none').click();
```

Comment:
How is this working? you intercept the network request using `cy.intercept(),` How will your test case know when this request is made and proceed to the next? What if this call takes longer

Reply:
We need this method because the API sometimes fails in the headless browser. Using cy.intercept() and cy.wait(), we can make sure the test waits for a successful API response before proceeding